src/styles/card.scss

```diff
@@ -0,0 +1,102 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  background-color: $white;
+  border: 1px solid $light-grey;
+  border-radius: 5px;
+  transition: all 0.3s ease;
```

Comment:
It is a good practice to specify the transition property
https://stackoverflow.com/questions/8947441/css3-transitions-is-transition-all-slower-than-transition-x

Reply:
done